recipe/meta.yaml

```diff
@@ -1,26 +1,25 @@
 {% set name = "croniter" %}
-{% set version = "1.0.15" %}
+{% set version = "1.3.7" %}
 {% set bundle = "tar.gz" %}
 {% set hash_type = "sha256" %}
-{% set hash = "a70dfc9d52de9fc1a886128b9148c89dd9e76b67d55f46516ca94d2d73d58219" %}
+{% set hash = "72ef78d0f8337eb35393b8893ebfbfbeb340f2d2ae47e0d2d78130e34b0dd8b9" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
```

Comment:
It should be `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/croniter-{{ version }}.tar.gz`.
Because `https://files.pythonhosted.org` is not a great solution.

Reply:
The reason why that link was used was to solve the issue with the linter.

Reply:
Revert the `URL` back to the initial state regardless of the Linter results: 
https://github.com/AnacondaRecipes/croniter-feedstock/blob/2323958ee48997dbda07dc2104293a15d250dba4/recipe/meta.yaml#L13